### PR TITLE
continuous integration

### PR DESCRIPTION
**Pull request by pheel**

### CODE REVIEW

**Comment by znat** [comment]
(at least) some of the tests are failing because meteor methods can't be invoked directly on the prod build. Back to unit tests for methods @MatthieuJnon

File: cli/src/cli.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ program
 
 program
     .command('logs')
-    .option('-v, --verbose', 'Display Docker Compose start-up logs')
```
Comment (znat) [inline cli/src/cli.js]: Why did you remove the verbose option?

File: cli/src/commands/services.js [hunk 2]
```diff
@@ -27,7 +27,8 @@ export async function dockerComposeUp({ verbose = false }, workingDir, spinner =
     try{
         startSpinner(spinner, 'Starting Botfront...')
         await shellAsync(command, { silent: !verbose });
-        await waitForService('botfront');
+        if (spinner) { await waitForService('botfront'); }
+        else { await wait(20000); process.exit(0); }
```
Comment (znat) [inline cli/src/commands/services.js]: Why don't you just wait for the service to be available?

File: cli/src/cli.js (hunk 1 above)
Comment (pheel) [reply cli/src/cli.js]: It wasn,t actually used.

File: cli/src/commands/services.js (hunk 2 above)
Comment (pheel) [reply cli/src/commands/services.js]: Could not make it run on GCS (tried a bunch).
